package.json

```diff
@@ -63,6 +66,7 @@
     "@babel/polyfill": "^7.12.1",
     "@babel/preset-env": "^7.12.11",
     "@commitlint/config-conventional": "^16.2.4",
+    "@rollup/plugin-image": "^2.1.1",
```

Comment:
To support `woff`, I had to set up file imports, so I went ahead and got rid of `rootPath` while I was at it.